terraform/postgresql/provision/main.tf

```diff
@@ -71,6 +71,10 @@ resource "aws_db_instance" "db_instance" {
   max_allocated_storage       = local.max_allocated_storage
   storage_encrypted           = var.storage_encrypted
   deletion_protection         = var.deletion_protection
+  backup_retention_period     = var.backup_retention_period
+  backup_window               = var.backup_window == "00:00-00:00" ? null : var.backup_window
```

Comment:
I was wondering if using null default and adding the nullable property would add more readability, but I understand that you have already considered this possibility and have opted for this option